modules/core/src/lib/deck.ts

```diff
@@ -960,6 +968,12 @@ export default class Deck {
 
     this.deckPicker = new DeckPicker(gl);
 
+    this.widgetManager = new WidgetManager({
+      deck: this,
+      parent: this.canvas?.parentElement
```

Comment:
Is this better than the `parent` prop since it ensures it's right on top of the canvas?

Reply:
`deck.parent` comes from user props and is not always assigned. I think this part of the code needs an API audit but that's out of the scope of this PR.

Reply:
Should we call it `parentElement` to emphasize that it is a DOM element?